src/saksoversikt/Saksoversikt.tsx

```diff
@@ -14,7 +14,7 @@ import {AlertStripeAdvarsel} from "nav-frontend-alertstriper";
 import SaksoversiktIngenSoknader from "./SaksoversiktIngenSoknader";
 
 const Saksoversikt: React.FC = () => {
-    document.title = "Økonomisk sosialhjelp";
+    document.title = "Dine søknader - Økonomisk sosialhjelp";
```

Comment:
Mangler denne useBannerTittel()?

Reply:
Dette er forsiden på innsyn, og den har en litt annen banner. Blir nok litt mer tweaking på disse overskriftene etterhvert, men nå skal det i hvert fall være up-to-date med tanke på UU.

Reply:
Topp 👍🏻